macro-plantuml-macro/src/main/java/org/xwiki/contrib/plantuml/PlantUMLGenerator.java

```diff
@@ -46,6 +67,11 @@ public interface PlantUMLGenerator
      * @throws IOException when there's a generation or writing error
      * @since 2.4
      */
-    void outputImage(String input, OutputStream output, String serverURL, PlantUMLDiagramFormat format)
-            throws IOException;
+    default void outputImage(String input, OutputStream output, String serverURL, PlantUMLDiagramFormat format)
+            throws IOException
+    {
+        // TODO: After transition period, this default method implementation must be removed.
+        // Default method implementation is for backward compatibility only.
+        outputImage(input, output, serverURL);
+    }
```

Comment:
I don't understand the comment. Why is this method removed since you're adding it (with a new parameter)?

Reply:
I see it's called below at https://github.com/xwiki-contrib/macro-plantuml/pull/5/commits/07820646fd491790b5f3d4eafac02bde21917f40#diff-907f4f8421589200188f94541c54f3baaff5a28c75fb96ee6ab890a012ff1789R125